# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../clockspeed/0006-cross-compile.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 4cc77d2f2b6fe10cbce304dc2e551aeecb9e2de4 Mon Sep 17 00:00:00 2001
From: Christian Wiese <christian.wiese@securepoint.de>
Date: Thu, 29 Sep 2011 15:05:49 +0200
Subject: [PATCH 6/6] Makefile: fixed cross-compiling

---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index c9baad6..11b7e48 100644
--- a/Makefile
+++ b/Makefile
@@ -21,7 +21,7 @@ compile auto-str.c substdio.h readwrite.h exit.h
 
 auto_home.c: \
 auto-str conf-home
-	./auto-str auto_home `head -n 1 conf-home` > auto_home.c
+	echo -e "#include \"auto_home.h\"\nchar auto_home[]=\"`head -n 1 conf-home`\";" > auto_home.c
 
 auto_home.o: \
 compile auto_home.c
@@ -153,7 +153,7 @@ trymkffo.c compile load
 
 hasrdtsc.h: \
 tryrdtsc.c compile load
-	( ( ./compile tryrdtsc.c && ./load tryrdtsc && ./tryrdtsc \
+	( ( ./compile tryrdtsc.c && ./load tryrdtsc \
 	) >/dev/null 2>&1 \
 	&& echo \#define HASRDTSC 1 || exit 0 ) > hasrdtsc.h
 	rm -f tryrdtsc.o tryrdtsc
-- 
1.6.6.2

